atecc608a/main.c

```diff
@@ -410,9 +342,41 @@ psa_status_t test_hash_sha256()
     return status;
 }
 
+void setup_key_attributes(psa_key_attributes_t *attributes,
+                          psa_key_slot_number_t slot, int is_private)
+{
+    psa_set_key_slot_number(attributes, slot);
```

Comment:
Ideally we wouldn't hard code the slot number. We shouldn't encourage hard coding the slot number in our examples.

Reply:
There's a use case for that - if the user wants to use a certain pre-provisioned key, he/she should be able to point to it somehow.

Reply:
Yes. We should probably split those examples at some point.